### ISSUE TITLE
Feature Request: Sort, Limit, Offset

### ISSUE THREAD

**Issue by CinKon** [revolver-app/vuex-orm#26]
I think it woul be a huge improvement, if vuex-orm could provide the following functions:

- `limit`
- `offset`
- `sortBy`
- `sortDir`

I hope you can see my motivation behind this :)
If you have a large amount of entities, you might want to paginate them. To be able to do this nicely with vuex-orm, you'd need the functions above.

**Comment by kiaking** [comment]
You are completely right. We should add these.

Vuex ORM do have orderBy method right now. (and I thought I documented this and went to check the URL, then found out it wasn't... damn... sorry about that).

You can do this.

```js
// orderBy defaults to 'asc' if there are no 2nd argument.
store.getters['entities/users/query']().orderBy('age', 'desc').get()
```

Is this same as `sortBy` and `sortDir` (Dir stands for direction?)? Or is those different kind of features?

**Comment by CinKon** [comment]
Yeah, of course `orderBy` is the more common term. 
And yeah, `dir` stands for `direction` ;)

Is it possible to chain the `orderBy`-Method? So I could say

```javascript
store.getters['entities/users/query']().orderBy('name', 'desc').orderBy('age', 'desc').get()
```

To sort the entities by name, and if the name is equal -> by age?

I'd really appreciate the `limit` and `offset`-functions ;)

**Comment by mastermunj** [comment]
Yes, chaining `orderBy` is possible.

Reference: https://github.com/revolver-app/vuex-orm/blob/508e5edc4c025638376a4db3fbb6d6c345651e94/test/unit/Repo_Retrieve.spec.js#L315-L318

**Comment by mastermunj** [comment]
Created PR #27 for offset & limit. Hopefully it caters the need. 😊

**Comment by kiaking** [comment]
@CinKon 

> Is it possible to chain the orderBy-Method? So I could say

As @mastermunj mentioned, yes you can! It will work as you would expected 🚀

**Comment by kiaking** [comment]
Thanks to @mastermunj, Limit & Offset feature is now available at Vuex ORM 0.12.0 🎉 

The documentation is ready too!
https://revolver-app.gitbooks.io/vuex-orm/store/retrieving-data.html#order-by

Thank you all for your great support!
